app/services/signbank_export.rb

```diff
@@ -0,0 +1,68 @@
+class SignbankExport
+  include Rails.application.routes.url_helpers
+  class URLForbiddenCharacterError < StandardError; end
+
+  SEPARATOR = "|".freeze
+  QUERY = <<~SQL.squish.freeze
+    SELECT
+      array_to_string(array_agg(DISTINCT videos.id), '|') AS videos,
```

Comment:
nitpick: Maybe nicer to place all the array aggregations at the end so the 'primary' columns are shown in query results

Reply:
This is a great way to do this by the way